components/AddForm.js

```diff
@@ -0,0 +1,70 @@
+import CheckIcon from '../public/assets/check.svg';
+import styled from 'styled-components';
+
+export default function AddForm({ onAddItem, setAddActive }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const form = event.target.elements;
+    const data = {
```

Comment:
```suggestion
    const newItem = {
```
Better to rename this constant as "data" is very generic. Try and choose names that will help to explain what is happening :)